src/wallet/redux/actions/external.ts

```diff
@@ -18,11 +13,19 @@ export const FUNDING_FAILURE = 'WALLET.FUNDING.FAILURE';
 
 export const fundingRequest = (
   channelId: string,
-  state: WaitForFundingA | WaitForFundingB
+  myAddress: string,
```

Comment:
this could (should?) be
```
channelId: string,
engineSetupArgs: WalletEngineSetupArgs
```

Reply:
I think it should stay this way, a funding request shouldn't need to know about engineArgs